processing/sql/dml/011_prg_pa_update.sql

```diff
@@ -0,0 +1,22 @@
+-- okreslenie nazwy ulic
+-- zgodnosc simc, kod nazwy ulicy, nazwa ulicy
+update prg.pa as pa
+set teryt_ulica = trim(concat(cm.m, ' ', (u1.nazwa_2 || ' '), u1.nazwa_1)), teryt_ulic = u1.sym_ul
+from teryt.ulic as u1, teryt.cecha_mapping cm,
+(select simc.woj, simc.pow, simc.gmi, simc.sym as sym_czesc, simc.sym, simc.nazwa
+from teryt.simc
+where rodz_gmi not in ('8', '9')
+union
+select simc.woj, simc.pow, simc.gmi, x.sym, simc.sym as sym_czesc, simc.nazwa
```

Comment:
nie do końca rozumiem czemu sym i sym_czesc mają tutaj odwrotną kolejność?

Reply:
W tym konkretnym przypadku kolejność tych dwóch atrybutów jest bez znaczenia, bo obydwa brane są z tego samego pola (simc.sym). Ale poprawie to, aby było jednolicie. 
Znaczenie tego zapytania wyjaśnię w komentarzu do całego pull requesta